docs/ref/django-admin.txt

```diff
@@ -984,6 +984,12 @@ By default, the SQL created is for running the migration in the forwards
 direction. Pass ``--backwards`` to generate the SQL for
 unapplying the migration instead.
 
+.. versionchanged:: 1.9
```

Comment:
If you want to keep this note after `.. versionchanged:: 1.9` is removed, you should reformat it so we don't have to deindent it at that time. I'm not sure it's really important to document this behavior though.